Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hmcts/cnp-java-base:openjdk-8u181-jre-alpine3.8-1.0
+FROM hmcts/cnp-java-base:openjdk-8u191-jre-alpine3.9-1.0
```

Comment:
this wont exactly work ... its a major change.

See https://github.com/hmcts/cmc-claim-store/pull/733
or https://github.com/hmcts/probate-submit-service/pull/124